docs/quickstarts/express.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Express Quickstart
+description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your Express server.
+---
+
+<TutorialHero
+  framework="express"
+  exampleRepo={[
+    {
+      title: "Express Quickstart Repo",
+      link: "https://github.com/clerk/clerk-express-quickstart"
+    }
+  ]}
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "Create a Express application",
+      link: "https://expressjs.com/en/starter/installing.html",
+      icon: "expressjs",
+    }
+  ]}
+>
+  - Install `@clerk/express`
+  - Set your Clerk API keys
+  - Add `clerkMiddleware()` to your application
+  - Access auth state using `getAuth()`
+</TutorialHero>
+
+Learn how to integrate Clerk into your Express backend for secure user authentication and management. This guide uses TypeScript and allows you to choose your frontend framework.
+
+<Steps>
+  ### Install `@clerk/express`
+
+  Add Clerk's Express SDK to your project:
+
+  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: 'terminal' }}
+    npm install @clerk/express
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    yarn add @clerk/express
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    pnpm add @clerk/express
+    ```
+  </CodeBlockTabs>
+
+  ### Set your Clerk API keys
+
+  <SignedIn>
+    Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+  </SignedIn>
+
+  <SignedOut>
+    1. Navigate to the Clerk Dashboard.
+    1. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+    1. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+    1. Paste your keys into your `.env.local` file.
+
+    The final result should resemble the following:
+  </SignedOut>
+
+  ```sh {{ filename: '.env.local' }}
+  CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+
+  This guide uses `dotenv` to load the environment variables. Install the package by running the following command:
+
+  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: 'terminal' }}
+    npm install dotenv
+    npm install -D @types/dotenv
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    yarn add dotenv
+    yarn add -D @types/dotenv
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    pnpm add dotenv
+    pnpm add -D @types/dotenv
+    ```
+  </CodeBlockTabs>
+
+  ### Add middleware to your application
+
+  The `clerkMiddleware` can be registered globally or for specific routes. This examples registers the middleware globally.
```

Comment:
```suggestion
  ### Add middleware to your application

  The [`clerkMiddleware()`](/docs/references/express/overview#clerk-middleware) helper integrates Clerk authentication into your Express application. It can be registered globally or for specific routes. This following example registers it globally.
```